### PR TITLE
Added Python Tutor, an interactive multi-language visualizer.

### PR DESCRIPTION
**Pull request by itsmed**
Added links for Python Tutor, an interactive multi-language visualization tool. Added links for Java, JavaScript, Python, TypeScript, and  Ruby. C and C++ links not added as visualization is not yet available.

### CODE REVIEW

**Comment by onebree** [comment]
- I would not necessarily count this as a tutorial...
- Change the link name to "(Language) Tutor", as that's what the page says when you visit it
- Remove the `#edit-mode` from the URLs -- leave it to the site to decide to add that or not

**Comment by itsmed** [comment]
Thanks for the input. I've implemented your ideas and submitted another pull request.

File: free-programming-interactive-tutorials-en.md [hunk 1]
```diff
@@ -39,7 +39,6 @@
 
 * [Learn C](http://www.learn-c.org)
 
-
```
Comment (onebree) [inline free-programming-interactive-tutorials-en.md]: This line should not be removed. Please add it back

**Comment by itsmed** [comment]
I replaced the accidentally removed line.

**Pull request merged**